test/react.js

```diff
@@ -42,6 +42,16 @@ const createMiddleEnd = (init = true, initialState = 0) => {
 
 describe('React', () => {
 
+    let restoreConsole;
+    before(() => {
+
+        restoreConsole = TestingHooks.suppressErrorOutput();
+    });
+    after(() => {
+
+        restoreConsole();
+    });
```

Comment:
A React error log was surfacing in-console during test runs with the latest version of `@react-test-renderer/testing-hooks` (specifically the first React test `throws if user does not provide a middle-end`). Not sure what exactly introduced that change, but I thought this would be a fine enough workaround, assuming we'd never want any React error logging to surface

Specifics if helpful:

- suppressErrorOutput definition: https://github.com/testing-library/react-hooks-testing-library/blob/f480f74d9e4067be248aff69abac50768de717ce/src/core/console.ts#L3
- the error logged:

```sh
The above error occurred in the <TestComponent> component:

    at TestComponent (/strange-middle-end/node_modules/@testing-library/react-hooks/lib/helpers/createTestHarness.js:20:5)
    at Suspense
    at ErrorBoundary (/strange-middle-end/node_modules/react-error-boundary/dist/react-error-boundary.cjs.js:40:35)
```